src/analysis/retail/rogue/shared/talents/ColdBlood.tsx

```diff
@@ -0,0 +1,109 @@
+import { formatNumber } from 'common/format';
+import TALENTS from 'common/TALENTS/rogue';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import { calculateMaxCasts } from 'parser/core/EventCalculateLib';
+import Events, { DamageEvent } from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+import { ThresholdStyle } from 'parser/core/ParseResults';
+import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+class ColdBlood extends Analyzer {
+  static dependencies = {
+    abilities: Abilities,
+  };
+  casts: number = 0;
+  cooldown: number = 90;
+  damage: number = 0;
+  lashDamage: number = 0;
+  maxCasts = 0;
+  protected abilities!: Abilities;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.COLD_BLOOD_TALENT);
+    if (!this.active) {
+      return;
+    }
+
+    this.addEventListener(
+      Events.damage.by(SELECTED_PLAYER).spell(TALENTS.COLD_BLOOD_TALENT),
+      this.onDamage,
+    );
+
+    this.addEventListener(Events.fightend, this.adjustMaxCasts);
+
+    (options.abilities as Abilities).add({
+      spell: TALENTS.COLD_BLOOD_TALENT.id,
+      category: SPELL_CATEGORY.COOLDOWNS,
+      cooldown: this.cooldown,
+      gcd: {
+        base: 1000,
+      },
```

Comment:
this should probably be a static gcd

```suggestion
      gcd: {
        static: 1000,
      },
```